binary-search-trees/03-Binary-Search-Trees-Traversal.md

```diff
@@ -1,4 +1,4 @@
-# Traversals
+# Serializing with Tree Traversal
```

Comment:
I'm unable to comment on the specific line as it hasn't been changed in this PR, but on line 261 of this file, there's a broken link to a lesson about queues. I might suggest replacing the link to an article like [GeeksForGeeks](https://www.geeksforgeeks.org/queue-data-structure/) or [Tutorialpoint](https://www.tutorialspoint.com/data_structures_algorithms/dsa_queue.htm) since I can't find a lesson that goes over queues in this curriculum.